Mercury-Store/Mercury-Store/Profile/Models/ProfileModel.swift

```diff
@@ -6,3 +6,33 @@
 //
 
 import Foundation
+
+struct Customers {
+    let customers: [Customer]
+}
+
+// MARK: - Customer
+struct Customer  {
+    let id: Int
+    let email: String
+    let createdAt, updatedAt: Date
+    let firstName, lastName , tags: String
+    let ordersCount: Int
+    let totalSpent: String
+    let lastOrderID: Int?
+    let verifiedEmail: Bool
+    let phone: String
+    let lastOrderName: String?
+    let addresses: [Address]
+    let defaultAddress: Address
+}
+struct Address  {
+    let id, customerID: Int
+    let firstName, lastName: String
+    let company: NSNull
```

Comment:
This is auto generated and should be avoided, please check API documentation for the exact type/ model.
```suggestion
    let company: // String? / Int? 
```